src/modules/core/components/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -1,16 +1,31 @@
 import React from 'react';
-import MenuLine from './MenuLine';
+import classnames from 'classnames';
 
 import styles from './HamburgerMenu.css';
 
-const HamburgerMenu = () => {
+const displayName = 'HamburgerMenu';
+
+interface Props {
+  isOpen: boolean;
+}
+const HamburgerMenu = ({ isOpen }: Props) => {
+  const MenuLine = () => (
+    <div
+      className={classnames(styles.menuLine, {
+        [styles.menuOpen]: isOpen,
+      })}
+    />
+  );
+
   return (
     <div className={styles.container}>
```

Comment:
As a rule of thumb, root classes for a component are called `main`.

```suggestion
    <div className={styles.main}>
```